common.c

```diff
@@ -1101,6 +1102,27 @@ trace_mode(mode_t mode, char *p)
 	*p = '\0';
 }
 
+/* printf backtrace callback */
+void trace_printf_backtrace(void)
+{
+	void *callstack[128];
+	int old_trace_state;
+
+	old_trace_state = trace_disable();
+
+	int i, frames = backtrace(callstack, 128);
+	char **strs = backtrace_symbols(callstack, frames);
```

Comment:
Currently it disabled tracing before calling backtrace, so it would not be in infinite loop.
It's working with testing codes.
Are you getting infinite looping issue?